examples/scenarioSatelliteConstellation.py

```diff
@@ -0,0 +1,292 @@
+uses the stand alone Basilisk architecture rather than using the ''examples/FormationBskSim`` or ``examples/MultiSatBskSim`` 
+architectures for simultaneously simulating multiple spacecraft.
+
+The script is found in the folder ``basilisk/examples`` and executed by using::
+
+    python3 scenarioSatelliteConstellation.py
+
+Illustration of Simulation Results
+----------------------------------
+
+::
+
+    show_plots = True, a = 29994000, i = 56, T = 24, P = 3, F = 1
+
+The default scenario is modelled after the Galileo constellation with an altitude of 29,994 km, inclination of 
+56 degrees, and 24 satellites across 3 planes with a phasing offset of 1.
+
+.. image:: /_images/Scenarios/scenarioSatelliteConstellationGroundTrack.svg
+   :align: center
+
+Simulation Visualization In Vizard
+----------------------------------
+
+To add a visualization of each spacecraft's Earth nadir pointing, a cone algined with the pointing direction 
+is created through the ``vizInterface``::
+
+    vizSupport.createConeInOut(viz, fromBodyName=scList[i].ModelTag, toBodyName='earth', coneColor='teal',
+                               normalVector_B=[1, 0, 0], incidenceAngle=30/macros.D2R, isKeepIn=False,
+                               coneHeight=10.0, coneName='pointingCone')
+
+.. image:: /_images/Scenarios/scenarioSatelliteConstellationVizard.svg
+   :align: center
+
+"""
+
+#
+# Basilisk Scenario Script and Integrated Test
+#
+# Purpose:  Integrated test of the spacecraft() and gravity modules.  Illustrates
+#           a 3-DOV spacecraft on a range of orbit types.
+# Author:   Andrew Morell
+# Creation Date:  Dec. 13, 2023
+#
+
+import os
+from copy import copy
+
+import matplotlib.pyplot as plt
+import numpy as np
+# To play with any scenario scripts as tutorials, you should make a copy of them into a custom folder
+# outside of the Basilisk directory.
+#
+# To copy them, first find the location of the Basilisk installation.
+# After installing, you can find the installed location of Basilisk by opening a python interpreter and
+# running the commands:
+from Basilisk import __path__
+
+bskPath = __path__[0]
+fileName = os.path.basename(os.path.splitext(__file__)[0])
+
+# Copy the folder `{basiliskPath}/examples` into a new folder in a different directory.
+# Now, when you want to use a tutorial, navigate inside that folder, and edit and execute the *copied* integrated tests.
+
+
+# import simulation related support
+from Basilisk.simulation import (spacecraft, ephemerisConverter)
+# general support file with common unit test functions
+# import general simulation support files
+from Basilisk.utilities import (SimulationBaseClass, macros, orbitalMotion,
+                                simIncludeGravBody, unitTestSupport, vizSupport)
+from Basilisk.fswAlgorithms import locationPointing
+from Basilisk.simulation import (simpleNav, planetEphemeris)
+
+# always import the Basilisk messaging support
```

Comment:
This comment seems left over from a copy and paste. Worth checking that every comment in this file is correct and relevant.

Reply:
I removed several un-needed comments in my modifications.